.github/workflows/rebuild-100k.yml

```
@@ -0,0 +1,57 @@
+name: Rebuild 100k sample
+
+on:
+  # cron job once a week on mondays
+  schedule:
+    - cron:  '* * * * 1'
+  # Manually triggered using GitHub's UI
+  workflow_dispatch:
+
+jobs:
+  gisaid:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: nextstrain/.github/actions/setup-nextstrain-cli@master
+      with:
+        python-version: 3.9
```

```suggestion
        python-version: 3.10
```

Should work with 3.10 IIRC

---

Done :)